src/screens/Followings/elements/TabBar/FollowingTabBar.js

```diff
@@ -4,6 +4,7 @@ import {StyleSheet, TouchableOpacity, View} from 'react-native';
 
 import {colors} from '../../../../utils/colors';
 import {fonts} from '../../../../utils/fonts';
+import {COLORS} from '../../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the theme file is a good change as it centralizes the color definitions and makes the code more maintainable. However, please ensure that all other color references in this file are also updated to use the `COLORS` object instead of the old `colors` object.
```diff
- import {colors} from '../../../../utils/colors';
+ import {COLORS} from '../../../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->